# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# feedback message
# here you might define some intermediate feedback message that reports about the current status of computation -> useful if the handling of the request might take several seconds or longer
float32 percentageDone	# this is just a number telling how far the action accomplished the task (in the program the computations are delayed to show the effect)

